3.10/arangosearch-ranking.md

```diff
@@ -53,6 +53,67 @@ Scoring functions cannot be used outside of `SEARCH` operations, as the scores
 can only be computed in the context of a View, especially because of the
 inverse document frequency (IDF).
 
+### Stable pagination for results
+
+The `SORT` operation does not guarantee a stable sort if there is no unique value
+to sort by. This leads to an undefined order when sorting equal documents.
+
+To achieve stable pagination, you must meet the following requirements: 
+- the dataset should not be changed
+- the `SORT` operation must have at least one field with an unique value to sort by
```

Comment:
Sentence style?
```suggestion
- The dataset should not be changed.
- The `SORT` operation must have at least one field with an unique value to sort by.
```
Should we point out that even if the dataset doesn't change that you can get an unstable order with pagination? It's not so clear what the first point is supposed to say.